go.mod

```diff
@@ -4,7 +4,7 @@ go 1.22.10
 
 require (
 	github.com/alexliesenfeld/health v0.8.0
-	github.com/ava-labs/avalanchego v1.12.0-initial-poc.9.0.20241122192639-7c3ad181c928
+	github.com/ava-labs/avalanchego v1.12.1-0.20241203223023-13e07a1b3a1e
```

Comment:
Let's hold off on merging until https://github.com/ava-labs/icm-services/pull/580 is merged to `master`.

Reply:
https://github.com/ava-labs/avalanchego/pull/3573 has been merged